actionmailer/test/test_helper_test.rb

```diff
@@ -207,6 +219,37 @@ def test_assert_no_enqueued_emails_failure
 
     assert_match(/0 .* but 1/, error.message)
   end
+
+  def test_assert_enqueued_email_with
+    assert_nothing_raised do
+      assert_enqueued_email_with TestHelperMailer, :test do
+        silence_stream($stdout) do
+          TestHelperMailer.test.deliver_later
+        end
+      end
+    end
+  end
+
+  def test_assert_enqueued_email_with_args
+    assert_nothing_raised do
+      assert_enqueued_email_with TestHelperMailer, :test_args, args: ["some_email", "some_name"] do
+        silence_stream($stdout) do
+          TestHelperMailer.test_args("some_email", "some_name").deliver_later
+        end
+      end
+    end
+  end
+
+  def test_assert_enqueued_email_with_parameterized_args
+    assert_nothing_raised do
+      assert_enqueued_email_with TestHelperMailer, :test_parameter_args, args: { all: "good" } do
+        silence_stream($stdout) do
+          TestHelperMailer.with(all: "good").test_parameter_args.deliver_later
+          puts enqueued_jobs.inspect
```

Comment:
Why is this `puts` needed?

Reply:
oh god forgot that in there. I need to concentrate.